_posts/2016-01-13-state-of-the-lab.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: State of the Lab 2016
+author: whoburg
+---
+
+A recent e-mail I sent to the lab:
+
+<tt>
+> Folks:
+
+> Welcome back from the holidays and happy 2016. I thought last night's State of the Union address a good occasion to reflect on our first 1.4 years of existence and where we're headed.
```

Comment:
@pgkirsch, definitely a good question. Woody's response was just so inspiring that I found myself contributing an unnecessary two cents.